tests/device_tests/test_msg_sd_protect.py

```diff
@@ -31,6 +31,11 @@ def test_sd_protect(self, client):
         with pytest.raises(TrezorFailure):
             device.sd_protect(client, proto.SdProtectOperationType.DISABLE)
 
+        # skip the rest of the tests if SD card is not present
+        debug_state = client.debug.state()
+        if not debug_state.sd_card:
+            return
```

Comment:
I think we should show warning when skipping the test:
`warnings.warn("SD card not present. SD protect tests were skipped.")`

Reply:
Good idea.